package.json

```diff
@@ -75,7 +75,7 @@
     "exists-sync": "0.0.3",
     "fake-xml-http-request": "^1.4.0",
     "faker": "3.0.1",
-    "pretender": "^1.2.0",
+    "pretender": "^1.3.0",
```

Comment:
I think we can remove this change from this PR now? Once you do the tests will rerun and we can investigate any errors.

Let's get this puppy merged!